### ISSUE TITLE
Admin panel/blueprint breaks indexer

### ISSUE THREAD

**Issue by codebude** [trilbymedia/grav-plugin-tntsearch#132]
I may have found a bug. The value for the stemmer setting "Default" should be `no`. (At least as far as I can see from code.)

For the dropdown in the admin panel the value for stemmer->default is defined here:
- https://github.com/trilbymedia/grav-plugin-tntsearch/blob/9299fec6c98dd40f4e0d18a2bd85c0ba3119177d/blueprints.yaml#L184
- https://github.com/trilbymedia/grav-plugin-tntsearch/blob/9299fec6c98dd40f4e0d18a2bd85c0ba3119177d/blueprints.yaml#L186

Unfortunately the 'no' gets rendered as `0` in the admin ui (check "data-value"):  
```html
<div class="selectize-input items full has-options has-items">
     <div class="item" data-value="0">Disabled</div>
     <input type="select-one" autocomplete="off" tabindex="" style="width: 4px;">
</div>
```

If you click on save in admin ui, the value `0` will be stored in `user/config/plugins/tntsearch.yaml` which ultimately lead to the following exeception when you start the indexer:  
```
Fatal error: Uncaught Error: Class "TeamTNT\TNTSearch\Stemmer\0Stemmer" not found in /var/www/html/user/plugins/tntsearch/vendor/teamtnt/tntsearch/src/Indexer/TNTIndexer.php:139
Stack trace:
#0 /var/www/html/user/plugins/tntsearch/classes/GravTNTSearch.php(237): TeamTNT\TNTSearch\Indexer\TNTIndexer->setLanguage('0')
// ...
```
 
To get an actual `no` into the config file via the admin ui, I propose to change the blueprint.yaml from  
```yaml
      default: no
      options:
        no: Disabled
```  
to

```yaml
      default: 'no'
      options:
        'no': Disabled
```

**Comment by Moonbase59** [comment]
Confirm that, using Grav  v1.7.43, Admin v1.10.43, TNT Search v3.4.0.

After having tried the German stemmer, I wanted to switch it off again and got a `'0'` in `user/config/plugins/tntsearch.yaml` as value for `stemmer`.

My quick-n-dirty solution was to look up the original in `user/plugins/tntsearch/tntsearch.yaml` and use `stemmer: default`. This showed "Disabled" in the Admin panel and worked for reindexing.

I also tried your solution (quote the `no`’s), which works fine here.

**Comment by codebude** [comment]
Glad to hear that you could solve your problem. Unfortunately there's no response from the package owner @trilby-bot 
This would be really easy to fix.

**Comment by rhukster** [comment]
@trilby-bot is just a bot, he's not going to be much help :)

I will look at this issue when i'm back next week.

**Comment by aasampaio** [comment]
Hi @rhukster !

Testing GRAV+TNT Search on two different VMs.
Both VMs have Grav v1.7.46 - Admin v1.10.46, TNT Search v3.4.0 + PHP 8.2

- VM1 (Debian GNU/Linux 11 (bullseye) running on Cloudways) - GRAV + TNT Search work out of the box box - no update to blueprints.yaml is needed!

- VM2 (Ubuntu 22.04.4 LTS on our corporate data-center) - Error while creating index when Stemmer is set to "disabled" - applied @codebude solution which corrected the issue.

No apparent differences in PHP config... Any ideas on why these two VMs behave differently? Can the OS difference be a factor here?
Maybe there are additional requirements for TNTSearch not listed in the docs?

**Comment by aasampaio** [comment]
Hi 

The error was `Class "TeamTNT\TNTSearch\Stemmer\0Stemmer" not found`.
This how it looked on the admin dashboard.
![image](https://github.com/user-attachments/assets/1f475ead-679d-48cb-9065-4a48455608e3)
Also got a similar error when indexing via command line:

**Comment by rhukster** [comment]
thanks for bringing this back to my attention, now fixed in develop branch

**Comment by aasampaio** [comment]
Thank you @rhukster. When can we expect this to make it into a release?

**Comment by rhukster** [comment]
Should be in latest release I made available 5 days ago: v3.4.1
